test/test_tree_2d_advection.jl

```diff
@@ -41,20 +42,23 @@ EXAMPLES_DIR = joinpath(pathof(Trixi) |> dirname |> dirname, "examples", "tree_2
     @test_trixi_include(joinpath(EXAMPLES_DIR, "elixir_advection_amr.jl"),
       # Expected errors are exactly the same as in the parallel test!
       l2   = [4.913300828257469e-5],
-      linf = [0.00045263895394385967])
+      linf = [0.00045263895394385967],
+      coverage_override = (maxiters=10^5,))
```

Comment:
Why do you use a big `maxiters` here?

Reply:
This AMR case covers most of the AMR code, but it has to run for a while to do so.